client/util/time.ts

```diff
@@ -2,27 +2,36 @@
 import lang from '../lang'
 
 export function secondsToTime(s: number): string {
-    const divide = [60, 24, 30, 12];
-    const unit = ['minute', 'hour', 'day', 'month'];
-    let time = s / 60;
+    const divide = [60, 24, 30, 12]
+    const unit = ['minute', 'hour', 'day', 'month']
+    let time = s / 60
 
-    const format = (key: string) => {
-        let tmp = time.toFixed(1)
+    const format = (key: string, u: number) => {
```

Comment:
Why are you passing `u` in as an argument? `format()` is a closure. It captures `time` from the upper scope.

Reply:
Because I can pass either `time` or `s` into it depending on which I want to do.
`time` doesn't play nice with the closure if it's below 60 seconds.